plugins/spu2-x/src/Windows/SndOut_XAudio2.inl

```diff
@@ -182,7 +181,7 @@ private:
             }
 
             EnterCriticalSection(&cs);
-            safe_delete_array(qbuffer);
+            m_buffer = nullptr;
```

Comment:
it should be useless (generally speaking). However still using an object in a thread and executing the destructor in another thread isn't nice.